src/test/csf/BasicNetwork.h

```diff
@@ -473,24 +470,20 @@ class BasicNetwork
     */
     template <class Period, class Rep>
     bool
-    step_for (std::chrono::duration<
-        Period, Rep> const& amount);
+    step_for(std::chrono::duration<Period, Rep> const& amount);
 };
 
 //------------------------------------------------------------------------------
 
 template <class Peer>
-BasicNetwork<Peer>::queue_type::queue_type(
-        qalloc const& alloc)
-    : alloc_ (alloc)
+BasicNetwork<Peer>::queue_type::queue_type(qalloc const& alloc) : alloc_(alloc)
```

Comment:
Nit: this line is longer than we usually allow. I'd put the initializer back on its own line.